app/src/main/java/com/github/dedis/student20_pop/PoPApplication.java

```diff
@@ -2,18 +2,26 @@
 
 import android.app.Application;
 import android.content.Context;
-
-import androidx.annotation.VisibleForTesting;
+import android.content.SharedPreferences;
+import android.util.Log;
 
 import com.github.dedis.student20_pop.model.Lao;
 import com.github.dedis.student20_pop.model.Person;
+import com.github.dedis.student20_pop.utility.security.PrivateInfoStorage;
+import com.google.gson.Gson;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
 
 public class PoPApplication extends Application {
```

Comment:
Just to be clear, I also made changes to this class but I'll make sure we have everything, and will modify the tests accordingly